src/collection.js

```diff
@@ -837,18 +907,18 @@ export default class Collection {
    *
    * @param  {KintoClient.Collection} client           Kinto client Collection instance.
    * @param  {SyncResultObject}       syncResultObject The sync result object.
+   * @param  {Object}                 changes          The change object.
+   * @param  {Array}                  changes.toDelete The list of records to delete.
+   * @param  {Array}                  changes.toSync   The list of records to create/update.
    * @param  {Object}                 options          The options object.
    * @return {Promise}
    */
-  async pushChanges(client, syncResultObject, options={}) {
+  async pushChanges(client, {toDelete=[], toSync}, syncResultObject, options={}) {
```

Comment:
Now I'm wondering if this new second argument shouldn't be split as two distinct ones. Thoughts?